### DISCUSSION TITLE
using integrated seurat objected inside ArchR

### DISCUSSION THREAD

**Discussion by dimurali93** [GreenleafLab/ArchR#1640] [Questions / Documentation]
Hi, 

We have 3 samples namely day3, day6 and week8 which are multiome observations comprising of SnATAC and ScRNA analysis. We have used seurat integration technique to combine these samples in scRNA. ArchR uses addClusters() call through which we can specify seurat as a method for clustering. We were wondering whether there is anyway to specify ArchR to use seurat integration as a method to generate clusters.
Thanks 
Divya

**Comment by rcorces** [comment]
Multiome data should be analyzed following the multiome tutorial.
https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html
